<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fzcalq</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="KHistoryComboBox" name="formulaBox"/>
      </item>
      <item>
       <widget class="QTextEdit" name="resultBox">
        <property name="font">
         <font>
          <family>Andale Mono</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="f7But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="f8But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="f9But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="f4But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="f5But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="f6But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="f1But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="f2But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="f3But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="f0But">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="dotBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="expBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>exp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="delBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>del</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ACBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="mulBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="addBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="f3But_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="ansBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>Ans</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="EXEBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>EXE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="divBut">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>253</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>KComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>formulaBox</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>returnSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f0But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f0Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f1But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f1Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f2But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f2Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f3But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f3Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f4But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f4Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f5But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f5Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f6But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f6Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f7But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f7Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f8But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f8Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f9But</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>f9Slot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dotBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dotSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>expBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>expSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ACBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ACSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>delSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>divBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>divSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mulBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mulSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>f3But_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>subSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EXEBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>EXESlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ansBut</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ansSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>returnSlot()</slot>
  <slot>f1Slot()</slot>
  <slot>f2Slot()</slot>
  <slot>f3Slot()</slot>
  <slot>f4Slot()</slot>
  <slot>f5Slot()</slot>
  <slot>f6Slot()</slot>
  <slot>f7Slot()</slot>
  <slot>f8Slot()</slot>
  <slot>f9Slot()</slot>
  <slot>f0Slot()</slot>
  <slot>dotSlot()</slot>
  <slot>expSlot()</slot>
  <slot>ACSlot()</slot>
  <slot>delSlot()</slot>
  <slot>mulSlot()</slot>
  <slot>divSlot()</slot>
  <slot>addSlot()</slot>
  <slot>subSlot()</slot>
  <slot>ansSlot()</slot>
  <slot>EXESlot()</slot>
 </slots>
</ui>
